build/all/Dockerfile

```diff
@@ -30,13 +30,17 @@ ARG KUSTOMIZE_VERSION=v5.0.1
 RUN wget https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz -O /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz && \
   tar -zxvf /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz -C /tmp && \
   mv /tmp/linux-amd64/helm /usr/local/bin/helm && \
+  mkdir -p ./vendor/helm.sh/helm/v3 && \
```

Comment:
I'm curious about the decision for this path, it seems perhaps misleading to put it at this vendor directory path given this is the license for a binary rather than a vendored dependency. Is this conventional?

Reply:
This whole PR is a hack to add the binary licenses to LICENSES.txt.

There are probably lots of alternative ways to do it, but this was expedient, preserves the sorted order of LICENSES.txt by delegating to the existing prepare licenses script, and is guaranteed not to conflict with any vendored dependencies. Even if we start depending on the root of the repo, the LICENSE file will probably still be identical.